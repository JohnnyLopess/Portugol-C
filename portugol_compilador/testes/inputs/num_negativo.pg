programa {
    inicio
        inteiro numeroNegativo = -1
        escreva(numeroNegativo)
}