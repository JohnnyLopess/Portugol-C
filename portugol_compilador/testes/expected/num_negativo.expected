#include <stdio.h>

int main() {
    int numeroNegativo = -1;
    printf("%d\n", numeroNegativo);
    return 0;
}